README.md

```diff
@@ -3,24 +3,36 @@ Repository for docker image files, for testing nix services.
 
 It will install systemd or upstart, along with SSH
 
-# NB The repository still needs to be moved and the images renamed.
-
-Look here for the latest images https://hub.docker.com/u/waffleimage
+Look here for the latest images https://hub.docker.com/u/litmusimage
 
 # Buildable images
-| Script | Supported OSes | Example Build Command |
-| ------------- | ------------- | ------------- |
-|  yum_initd_dockerfile | centos 6, oraclelinux 6, scientificlinux/sl 6 | 'docker build --rm --no-cache -t waffleimage/oraclelinux6 . -f yum_initd_dockerfile --build-arg BASE_IMAGE_TAG=6 --build-arg OS_TYPE=oraclelinux' 'docker build --rm --no-cache -t waffleimage/scientificlinux6 . -f yum_initd_dockerfile --build-arg BASE_IMAGE_TAG=6 --build-arg OS_TYPE=scientificlinux/sl' |
-|  yum_systemd_dockerfile | centos 7, oraclelinux 7, scientificlinux/sl 7 | 'docker build --rm --no-cache -t waffleimage/oraclelinux7 . -f yum_systemd_dockerfile --build-arg BASE_IMAGE_TAG=7 --build-arg OS_TYPE=oraclelinux' 'docker build --rm --no-cache -t waffleimage/scientificlinux7 . -f yum_systemd_dockerfile --build-arg BASE_IMAGE_TAG=7 --build-arg OS_TYPE=scientificlinux/sl' |
-|  apt_systemd_dockerfile | debian 8/9, ubuntu 16.04| docker build --rm --no-cache -t waffleimage/debian9 . -f apt_systemd_dockerfile --build-arg BASE_IMAGE_TAG=9 --build-arg OS_TYPE=debian |
-|  apt_initd_dockerfile | ubuntu 14.04| docker build --rm --no-cache -t waffleimage/ubuntu14.04 . -f apt_initd_dockerfile --build-arg BASE_IMAGE_TAG=14.04 --build-arg OS_TYPE=ubuntu |
-|  apt_sysvinit-utils_dockerfile | ubuntu 18.04 | docker build --rm --no-cache -t waffleimage/ubuntu18.04 . -f apt_sysvinit-utils_dockerfile --build-arg BASE_IMAGE_TAG=18.04 --build-arg OS_TYPE=ubuntu |
+| IMAGE | TAG | DOCKERFILE | BASE_IMAGE | BASE_IMAGE_TAG |
+| ------| ----| -----------| -----------| ---------------|
+| ubuntu | 14.04 | apt_initd_dockerfile | ubuntu | 14.04 |
+| ubuntu | 16.04 | apt_systemd_dockerfile | ubuntu | 16.04 |
+| ubuntu | 18.04 | apt_sysvinit-utils_dockerfile | ubuntu | 18.04 |
+| centos | 6 | yum_initd_dockerfile | centos | 6 |
+| centos | 7 | yum_systemd_dockerfile | centos | 7 |
+| centos | 8 | yum_systemd_dockerfile | centos | 8 |
+| scientificlinux | 6 | yum_initd_dockerfile | scientificlinux/sl | 6 |
+| scientificlinux | 7 | yum_systemd_dockerfile | scientificlinux/sl | 7 |
+| oraclelinux | 6 | yum_initd_dockerfile | oraclelinux | 6 |
+| oraclelinux | 7 | yum_systemd_dockerfile | oraclelinux | 7 |
+| debian | 8 | apt_systemd_dockerfile | debian | 8 |
+| debian | 9 | apt_systemd_dockerfile | debian | 9 |
+| debian | 10 | apt_sysvinit-utils_dockerfile | debian | 10 |
+
+# Build command
+`docker build --rm --no-cache -t litmusimage/$IMAGE:$TAG . -f $DOCKERFILE --build-arg BASE_IMAGE_TAG=$BASE_IMAGE_TAG --build-arg OS_TYPE=$BASE_IMAGE`
+
+## Example
+`docker build --rm --no-cache -t litmusimage/ubuntu:14.04 . -f apt_initd_dockerfile --build-arg BASE_IMAGE_TAG=14.04 --build-arg OS_TYPE=ubuntu`
```

Comment:
Suggest wrapping this in code blocks (couldn't use the suggestion tooling to show this, sorry!):

```
docker build --rm --no-cache -t litmusimage/ubuntu:14.04 . -f apt_initd_dockerfile --build-arg BASE_IMAGE_TAG=14.04 --build-arg OS_TYPE=ubuntu
```